src/delaunay.js

```diff
@@ -174,6 +174,9 @@ export default class Delaunay {
     return buffer && buffer.value();
   }
   renderPoints(context, r = 2) {
+    if (arguments.length === 1 && ["number", "string"].includes(typeof context)) {
+      [r, context] = [+context, null];
+    }
```

Comment:
I'd prefer simpler code that doesn't use `arguments` and redundant array allocations.

```suggestion
    if (typeof +context === 'number') {
      r = +context;
      context = null;
    }
```

Reply:
I don't think this would work, since +null and +undefined are "number".
What we can do is test for "number-like" with `x != null && +x === +x`